### PR TITLE
Change topologies option from 'notsystem' to 'application'

### PR DESCRIPTION
**Pull request by jpellizzari**
Fix for #2120 

A bug was introduced here: https://github.com/weaveworks/scope/commit/9833a854b1638d8bd3270323c8aa0041016ebb98#diff-6126bda540b57df79f551488091760daR56

To test:
1. Click the 'bug' icon in the bottom left of the screen
2. Click the 'Reset your local view state' option
3. When the page reloads, the 'Application' filter should be selected.

### CODE REVIEW

**Comment by jpellizzari** [comment]
@ekimekim Can you review please?

**Comment by jpellizzari** [comment]
@ekimekim Having some trouble getting integration tests to pass:

```
Launch scope and check it boots, with a spurious host arg
83ecbe3e7aeb8d55334bfd318fe86b45b9a7834afdce033592f2065e0697bf5c
Scope probe started
Weave Scope is reachable at the following URL(s):
  * http://10.240.0.20:4040/
Failed to find nodes weavescope after 60 secs
test #1 "curl -s http://host2-6126-0.us-central1-a.scope-integration-tests:4040/api/topology/containers?system=show | jq -r '[.nodes[] | select(.label == "weavescope")] | length'" failed:
	expected "1"
	got "0"
1 of 1 tests failed in 66.873s.
```

That failure doesn't look related to the change I made in this PR. Any thoughts on what might be causing that?

**Comment by ekimekim** [comment]
Hm. I haven't looked into it properly, but at a first glance, I think the following happened:
* Originally, option was system = `show` or `hide`. Test code above works.
* Backend is changed so option is `all`, `system` or `notsystem`. This causes the bug, and the default is still all. Therefore *because* of the bug, test code above still works.
* You fix bug. Test is now broken.

I think the fix is to change the argument to the curl url in that text to value `system`, but i'm not sure what the correct argument key is now? pretty sure it's no longer `system=`.

File: integration/config.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ has() {
     local host=$2
     local name=$3
     local count=${4:-1}
-    assert "curl -s http://${host}:4040/api/topology/${view}?system=show | jq -r '[.nodes[] | select(.label == \"${name}\")] | length'" "$count"
+    assert "curl -s http://${host}:4040/api/topology/${view}? | jq -r '[.nodes[] | select(.label == \"${name}\")] | length'" "$count"
 }
```
Comment (jpellizzari) [inline integration/config.sh]: @2opremio This ended up getting my tests to pass on this branch, but did I remove the purpose of the test?

Comment (ekimekim) [inline integration/config.sh]: Hm. Hard to say without carefully reading the whole thing, but I suspect the desired behaviour here is to "show all". So you should pass in the value corresponding to that. If that's the default, what you have here is fine (though nit: if you have no params, just omit the `?` entirely)

**Comment by jpellizzari** [comment]
@2opremio That change to from `s/containerLabelFiltersGroupID = "container-label-filters-group"/systemGroupID = "system"/` looks like it did the trick. Thanks!

**Pull request merged**